drivers/espi/espi_emul.c

```diff
@@ -29,6 +29,10 @@ struct espi_emul_data {
 	struct espi_cfg cfg;
 	/** List of eSPI callbacks */
 	sys_slist_t callbacks;
+#ifdef CONFIG_ESPI_PERIPHERAL_ACPI_SHM_REGION
+	/** ACPI Shared memory. */
+	uint8_t shm_acpi_mmap[CONFIG_ESPI_EMUL_HOST_ACPI_SHM_REGION_SIZE];
```

Comment:
I think the mmap should be stored on the host side.

Reply:
Done